time timestamp
string context
string target
string function
string method
bool flush_interaction
---
string result